database/postgresql/src/main/java/com/example/javaopencommerce/catalog/CatalogRepositoryImpl.java

```diff
@@ -4,6 +4,9 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
```

Comment:
Not a fan of that annotation. It would be better to create bean with a @Producer.